components/resume/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface Basics {
```

Comment:
Ended up adding types here and not later as I initially planned, and boy did these help when making a larger refactor, calling out the spots I was using wrong/old property names.